client/src/components/home/index.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @next/next/no-img-element */
+
+import { getHomeStats } from '@/types/generated/home-stat';
+
+import ImageSlider from '@/components/home/image-slider';
+import HomeNavBar from '@/components/home/nav-bar';
+
+import Footer from './footer';
+import HeroSection from './hero-section';
+import ModulesSection from './modules-section';
+import RecommendSection from './recommend-section';
+import SolutionsSection from './solutions-section';
+
+export default async function HomePage() {
+  const data = await getHomeStats({ populate: '*' });
+  const stats = data?.data?.map((item) => ({
+    title: item?.attributes?.title,
+    value: item?.attributes?.value,
+    class: item?.attributes?.class,
+  }));
```

Comment:
The `HeroSection` component is the only one using this data so maybe you could move this hook down the tree to avoid wider rerenders.